testgres/node.py

```diff
@@ -658,7 +658,7 @@ def safe_psql(self, dbname, query, username=None, input=None):
 
         return out
 
-    def dump(self, dbname, username=None, filename=None):
+    def dump(self, dbname, username=None, filename=None, formate=None):
```

Comment:
Probably should replace `formate` with `format` (I know, it's a function name, but who cares).

Reply:
done